onnxruntime/core/framework/node_unit.h

```diff
@@ -87,6 +89,7 @@ class NodeUnit {
   ProviderType GetExecutionProviderType() const noexcept;
 
   const Node& GetNode() const noexcept { return target_node_; }
+  const Node* GetActivationNode() const noexcept { return p_activation_node_; }
   const std::vector<const Node*>& GetDQNodes() const noexcept { return dq_nodes_; }
   const std::vector<const Node*>& GetQNodes() const noexcept { return q_nodes_; }
   std::vector<const Node*> GetAllNodesInGroup() const noexcept;
```

Comment:
I believe `GetAllNodesInGroup()` should be updated to also return the `p_activation_node_`. This is required in QNN EP, which uses this function in GetCapability() to add all nodes within a NodeUnit to the set of supported nodes.

Reply:
Good catch! Definitely needs to be included.